validator/src/cli.rs

```diff
@@ -1304,6 +1305,15 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 .long("replay-slots-concurrently")
                 .help("Allow concurrent replay of slots on different forks")
         )
+        .arg(
+            Arg::with_name("banking_trace_size")
+                .long("enable-banking-trace")
+                .value_name("MAX_BYTES")
+                .validator(is_parsable::<u64>)
+                .takes_value(true)
+                .default_value(&default_args.banking_trace_size)
+                .help("Write trace files for simulate-leader-blocks, retaining up to the maximum total bytes")
```

Comment:
total bytes across all files?

Reply:
yep, i tweaked the wording: db4732c1efec119e47e23e880bf2c7b8e1742d04